### DISCUSSION TITLE
Add data size sensor device class

### DISCUSSION THREAD

**Discussion by epenet** [home-assistant/architecture#833] [Entity Models] [answered]
I propose that we add a `data size` sensor device_class to home assistant core and frontend (+ documentation) with supported units:
- `***BITS`
- `***BYTES`
- `***BIBYTES`

Just looking at `DATA_MEGABYTES` unit, all these integrations would benefit: `filesize`, `folder`, `fully_kiosk`, `lidarr`, `netgear`, `nzbget`, `radarr`, `sabnzbd`, `synology_dsm`, `unifi`

Implementation PR: https://github.com/home-assistant/core/pull/83376

**Comment by frenck** [comment, marked as answer]
Makes sense, we have quite a few of those. 👍
